dask/dataframe/io/tests/test_parquet.py

```diff
@@ -135,7 +135,10 @@ def test_empty(tmpdir, write_engine, read_engine, index):
     ddf = dd.from_pandas(df, npartitions=2)
 
     ddf.to_parquet(fn, write_index=index, engine=write_engine)
-    read_df = dd.read_parquet(fn, engine=read_engine)
+    if index:
+        read_df = dd.read_parquet(fn, index='a', engine=read_engine)
```

Comment:
Maybe just `index=index`?

Reply:
`index` is a bool here, so `index=index` should fail (maybe I am missunderstanding?)